pytorch_lightning/trainer/trainer.py

```diff
@@ -219,7 +219,7 @@ def __init__(
             process_position: orders the progress bar when running multiple models on same machine.
 
             progress_bar_refresh_rate: How often to refresh progress bar (in steps). Value ``0`` disables progress bar.
-                Ignored when a custom callback is passed to :paramref:`~Trainer.callbacks`.
+                Ignored when a custom progress bar is passed to :paramref:`~Trainer.callbacks`. Default: 1
```

Comment:
this is confusing as you say default but in code is none, so maybe tell that it is set internally?

Reply:
what should I do?
we have other trainer args set as None and they are internally set to the defaults

Reply:
yes, but usually this is only done for one of the following reasons:

1.) We need to check for some type before setting the default (for example to raise deprecation warnings)
2.) They are mutable otherwise.
3.) They depend on other parameters
4.) They are really hard to parse/set correctly

Reply:
In general the 2) is most common

Reply:
Ok, here we are in case 3, because the default depends on an environment variable.
Then I will just update the docstring, or do you request other changes?

Reply:
updated docstring, do you like it better?